netbox/extras/tests/test_customvalidator.py

```diff
@@ -67,22 +53,6 @@ def validate(self, instance):
 
 class CustomValidatorTest(TestCase):
 
-    @override_settings(CUSTOM_VALIDATORS={'circuits.provider': [min_validator]})
-    def test_configuration(self):
-        self.assertIn('circuits.provider', settings.CUSTOM_VALIDATORS)
-        validator = settings.CUSTOM_VALIDATORS['circuits.provider'][0]
-        self.assertIsInstance(validator, CustomValidator)
-
-    @override_settings(CUSTOM_VALIDATORS={'circuits.provider': [min_validator]})
-    def test_min(self):
-        with self.assertRaises(ValidationError):
-            Provider(name='Provider 1', slug='provider-1', asn=1).clean()
-
-    @override_settings(CUSTOM_VALIDATORS={'circuits.provider': [max_validator]})
-    def test_max(self):
-        with self.assertRaises(ValidationError):
-            Provider(name='Provider 1', slug='provider-1', asn=65535).clean()
-
```

Comment:
These tests need to stay; they evaluate custom validation functionality. (We're just using the Provider model as an example.) I'll tweak them to use the ASN model instead.